PoExtractor.sln

```diff
@@ -7,13 +7,17 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PoExtractor.Core", "PoExtra
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PoExtractor.Core.Tests", "PoExtractor.Core.Tests\PoExtractor.Core.Tests.csproj", "{4E0AF6EC-9BD0-4D39-8123-06163362185D}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PoExtractor.CS", "PoExtractor.CS\PoExtractor.CS.csproj", "{8964B6F8-7548-43DB-8F63-230348A40A30}"
-EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PoExtractor.Liquid", "PoExtractor.Liquid\PoExtractor.Liquid.csproj", "{9DB27A54-F56F-465E-A1A9-28791C64710B}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PoExtractor.OrchardCore", "PoExtractor.OrchardCore\PoExtractor.OrchardCore.csproj", "{C81FF74F-5E1E-402E-A505-E4CEE495CF1A}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "PoExtractor", "PoExtractor\PoExtractor.csproj", "{D358C2C3-D568-4C60-BBFA-38AA2BE95C7A}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PoExtractor", "PoExtractor\PoExtractor.csproj", "{D358C2C3-D568-4C60-BBFA-38AA2BE95C7A}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PoExtractor.Languages", "PoExtractor.Languages\PoExtractor.Languages.csproj", "{7A43A348-20AD-4433-9A13-719439243BE7}"
```

Comment:
I might prefer _PoExtractor.DotNet_, _PoExtractor.DotNet.CS_ and _PoExtractor.DotNet.VB_ as the project names. These names align better with _PoExtractor.Liquid_ and also  _Languages_ might be confusing because translations are all about languages, but not the programming ones. What do you think? Does it make a sense?